app/controllers/projects_controller.rb

```diff
@@ -2,6 +2,7 @@ class ProjectsController < ApplicationController
 
   def index
     @projects = Project.order(updated_at: :desc).all
+    @review = Review.order("RANDOM()").first
```

Comment:
I think this might be rather expensive because it requires random shuffling all the reviews.

`@review = Review.find(Random.rand(Review.length))` 

Something like that? Not sure if the `Review.length` bit is right.